BlueDBOnDisk/src/main/java/org/bluedb/disk/collection/index/IndexManager.java

```diff
@@ -8,23 +8,22 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+
 import org.bluedb.api.exceptions.BlueDbException;
 import org.bluedb.api.index.BlueIndex;
 import org.bluedb.api.index.KeyExtractor;
 import org.bluedb.api.keys.BlueKey;
 import org.bluedb.api.keys.ValueKey;
-import org.bluedb.disk.collection.ReadOnlyBlueCollectionOnDisk;
+import org.bluedb.disk.collection.ReadWriteBlueCollectionOnDisk;
 import org.bluedb.disk.file.FileUtils;
 import org.bluedb.disk.recovery.IndividualChange;
 
-public class IndexManager<T extends Serializable> {
-
-	private static final String INDEXES_SUBFOLDER = ".index";
+public class IndexManager<T extends Serializable> extends ReadableIndexManager<T> {

```

Comment:
You should probably rename this class to ReadWriteIndexManager right?

Reply:
did so in the latest commit